src/stores/notifications/SpaceNotificationState.ts

```diff
@@ -74,7 +75,13 @@ export class SpaceNotificationState extends NotificationState {
 
         this._count = 0;
         this._color = NotificationColor.None;
-        for (const state of Object.values(this.states)) {
+        for (const [roomId, state] of Object.entries(this.states)) {
+            // We ignore unreads in LowPriority rooms, see https://github.com/vector-im/element-web/issues/16836
+            if (
+                Object.keys(this.rooms.find(r => r.roomId === roomId).tags).includes(DefaultTagID.LowPriority) &&
```

Comment:
You shouldn't be accessing `tags` on the room object directly, instead make use of `RoomListStore::getTagsForRoom` given that the RLS does some other funkiness atop `.tags`

Reply:
Ah, I didn't know that :+1: